src/main/java/healthy/lifestyle/backend/admin/users/repository/UserAdminRepository.java

```diff
@@ -0,0 +1,26 @@
+package healthy.lifestyle.backend.admin.users.repository;
+
+import healthy.lifestyle.backend.users.model.Country;
+import healthy.lifestyle.backend.users.model.Role;
+import healthy.lifestyle.backend.users.model.User;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface UserAdminRepository extends JpaRepository<User, Long> {
+    @Query("SELECT u FROM User u WHERE " + " u.role = :role AND "
```

Comment:
Let's see if it will work. Maybe you need `u.role.name = :role` where role is a String.

Concatenation of two strings can be replaced with one string: `"SELECT u FROM User u WHERE u.role = :role AND "`